packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -459,6 +442,114 @@ export class TablePageObject<T extends TableRecord> {
         }
     }
 
+    /**
+     * @param columnIndex The index of the column to the left of a divider being dragged. Thus, this
+     * can not be given a value representing the last visible column index.
+     * @param deltas The series of mouse movements in the x-direction while sizing a column.
+     */
+    public dragSizeColumnByRightDivider(
+        columnIndex: number,
+        deltas: number[]
+    ): void {
+        const divider = this.getColumnRightDivider(columnIndex);
+        if (divider === null) {
```

Comment:
Any reason not to simplify this condition?
```suggestion
        if (!divider) {
```